conf/install/interface.json

```diff
@@ -1 +1 @@
+{ "label" : "Interface", "expire_after": 86400, "meta_fields" : { "intf" : { "required" : 1, "ordinal" : 1 }, "node" : { "required" : 1, "ordinal" : 2 }, "alternate_intf": {}, "pop" : { "required" : 0, "fields" : { "name" : {}, "locality" : {}, "type" : {}}}, "circuit" : { "required" : 0, "fields" : { "name" : { "ordinal" : 1 }, "carrier_type" : {}, "customer_type" : {}, "role" : {}, "description" : { "ordinal" : 2 }, "type" : {}, "carrier" : {}, "customer" : {}, "owner" : {}, "circuit_id" : {}, "speed" : {} }, "classifier" : 1, "array" : 1 }, "service" : { "required" : 0, "classifier" : 1, "fields" : { "name" : { "ordinal" : 1 }, "description" : { "ordinal" : 2 }, "service_entity_type" : {}, "type" : {}, "entity" : {}, "contracted_bandwidth" : {}, "service_id" : {} }, "array" : 1 }, "entity" : { "fields" : { "name" : {}, "type" : {}, "contracted_bandwidth" : {}}, "required" : 0 }, "node_type" : {}, "node_role" : {}, "node_management_address" : {}, "network" : {}, "description" : { "ordinal" : 3 }, "type" : {}, "parent_interface" : {}, "tag" : {}, "max_bandwidth" : {}, "contracted_bandwidth" : {}, "interface_id" : {}}, "type" : "interface", "values" : { "input" : { "ordinal" : 1, "description" : "Input Bits/s", "units" : "bps" }, "output" : { "ordinal" : 2, "description" : "Output Bits/s", "units" : "bps" }, "inerror" : { "description" : "Input Errors/s", "units" : "pps" }, "outerror" : { "description" : "Output Errors/s", "units" : "pps" }, "status" : { "description" : "Status", "units" : "" }, "inUcast" : { "description" : "Input Packets/s", "units" : "pps" }, "outUcast" : { "description" : "Output Packets/s", "units" : "pps" }, "indiscard" : { "units" : "indiscard", "description" : "indiscard" }, "outdiscard" : { "units" : "outdiscard", "description" : "outdiscard" } } }
```

Comment:
This still has "service.service_entity_type". It should just be "service.entity_type" to be in line with the others. Make sure this change is coordinate with the meta manager code as well.